data/multiplayer/scenarios/2p_Dark_Forecast.lua

```diff
@@ -330,6 +330,10 @@ on_event("die", function()
 		message = _"The screams and pleas for mercy are finally silenced, as you remove your blood soaked blade from the last of the rebels. There will be no more resistance from the local scum. Your reign has finally earned stability.",
 		image ="wesnoth-icon.png",
 	}
+	wesnoth.achievements.set {
```

Comment:
I think this skips the notification, doesn't it?

Reply:
as discussed in `#development`, it does. Not sure what to do about it

Reply:
You could always call `wesnoth.wml_actions.set_achievement` instead, couldn't you?